app/assets/stylesheets/application.css.scss

```diff
@@ -202,3 +202,16 @@ main {
   color: white;
   box-shadow: 1px 1px 10px 1px rgba(0,0,0,0.7);
 }
+
+.town_table td{
+  border: none;
+  border-collapse: collapse;
+  border-spacing: 0px;
+  padding-bottom: 0px
```

Comment:
Declaration should be terminated by a semicolon<br>`0px` should be written without units as `0`